.github/ci-scripts/bash/functions-for-build

```diff
@@ -20,6 +20,24 @@ aws_configure_tessellops_artifacts() {
   aws configure set region ap-south-1
 }
 
+bash_build() {
+  set -e
+  DIR="${1}"
+  export PATH=$PATH:/home/github/.local/bin
```

Comment:
What binary is present here?

Reply:
Using common (`~/.local`) dir for virtualenv might cause issues (package conflicts, etc). Let us always have dedicated folder for virtualenv, within the cloned repo itself. So that once build is complete, the virtualenv gets cleaned up.

Reply:
User installed virtualenv module itself is getting there not a particular venv instance